spec/routing/tokens_controller_spec.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V1::TokensController, type: :routing do
```

Comment:
As far as i know, i most our Ruby on Rails project, we don't write test for the route.